### PR TITLE
Unbreak bleed on FreeBSD (also backport to prep-1908)

### PR DESCRIPTION
**Pull request by jbeich**
Closes #16361 (to avoid merge conflict)

Note, https://github.com/OpenRA/OpenAL-CS/pull/3 and https://github.com/OpenRA/SDL2-CS/pull/3 cannot be included until a new tag with those is created.

### CODE REVIEW

File: OpenRA.Platforms.Default/OpenRA.Platforms.Default.dll.config [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8" ?>
 <configuration>
-	<dllmap dll="freetype6" os="linux" target="libfreetype.so.6" />
+	<dllmap dll="freetype6" os="!windows" target="libfreetype.so.6" />
```
Comment (pchote) [inline OpenRA.Platforms.Default/OpenRA.Platforms.Default.dll.config]: `!windows` presumably also includes osx, which we won't want?

Comment (jbeich) [reply OpenRA.Platforms.Default/OpenRA.Platforms.Default.dll.config]: Last `dllmap` rule wins, so the next line with `os="osx"` will override the above.

File: Makefile [hunk 2]
```diff
@@ -151,7 +151,7 @@ test: core
 all: dependencies core
 
 core:
-	@command -v $(MSBUILD) >/dev/null || (echo "OpenRA requires the '$(MSBUILD)' tool provided by Mono >= 5.4."; exit 1)
+	@command -v $(word 1,$(MSBUILD)) >/dev/null || (echo "OpenRA requires the '$(MSBUILD)' tool provided by Mono >= 5.4."; exit 1)
```
Comment (pchote) [inline Makefile]: What does this do?

Comment (jbeich) [reply Makefile]: Replace `msbuild -verbosity:m -nologo` with `msbuild`.

Comment (jbeich) [reply Makefile]: Ironically, despite both FreeBSD sh and NetBSD sh being Almquist shell descendants only FreeBSD is affected.

**Comment by pchote** [comment]
Thanks! We definitely want this for the stable release too, so added to the milestone.
It will probably be a few days before I can look at the other PRs and test this in detail. Please ping me if I haven't followed up here by the weekend.

Adding the dependencies label so we don't forget to update the `fetch-thirdparty-deps` scripts before merging.

**Comment by pchote** [comment]
OpenRA/OpenAL-CS#3 and OpenRA/SDL2-CS#3 have been merged. Can you please now update `fetch-thirdparty-deps.sh` and `fetch-thirdparty-deps.ps1` to reference the `20190907` tags?

**Comment by jbeich** [comment]
Done. I've squashed the update into the commit touching `dllmap` in OpenRA repo to make `dllmap` change atomic.
```
$ gmake

$ fgrep linux **/*.config
<empty as intended>

$ diff <(fgrep -l dllmap **/*.config) <(fgrep -l openbsd **/*.config)
--- /tmp/zshiT61H7
+++ /tmp/zsh8PEoPD
@@ -1,4 +1,3 @@
-Eluant.dll.config
 OpenAL-CS.dll.config
 OpenRA.Platforms.Default.dll.config
 OpenRA.Platforms.Default/OpenRA.Platforms.Default.dll.config

$ mono --debug OpenRA.Game.exe Game.Mod=ra
<works fine>
```

**Review by pchote — APPROVED** [review]
LGTM :+1:

Tested a clean recompile and run on macOS, Ubuntu 18.04, Windows 10 without issue.

**Review by abcdefg30 — APPROVED** [review]

**Pull request merged**